docs/api/menu.md

```diff
@@ -30,6 +30,9 @@ effect on macOS.
 
 **Note:** This API has to be called after the `ready` event of `app` module.
 
+**Note:** Default menu is created automatically if the app does not set one.
+You can also set `process.noDefaultMenu` to `true` to disable this behavior.
```

Comment:
cc @MarshallOfSound

Reply:
I agree. “No”-prefixed options names are an anti-pattern. Also, wouldn’t this fit better on the `app` namespace?

Reply:
I've removed it completely for now. The default menu can be suppressed by calling `Menu.setApplicationMenu(null)`

Reply:
also the comment
> **Note:** This API has to be called after the `ready` event of `app` module.
does not seem accurate. `Menu.getApplicationMenu` can definitely be called before `ready`

Reply:
in that case `Menu.setApplicationMenu(null)` would be enough for suppressing the default menu. Otherwise something like `app.useDefaultMenu` would be necessary as the default menu would be created first and then immediately overwritten by the app menu, which is not efficient.